modules/perception/obstacle/onboard/camera_process_subnode.cc

```diff
@@ -91,37 +91,35 @@ bool CameraProcessSubnode::InitModules() {
 }
 
 void CameraProcessSubnode::ImgCallback(const sensor_msgs::Image &message) {
-  AdapterManager::Observe();
-  sensor_msgs::Image msg = AdapterManager::GetImageFront()->GetLatestObserved();
-
-  double timestamp = msg.header.stamp.toSec();
-
+  double timestamp = message.header.stamp.toSec();
+  ADEBUG << "CameraProcessSubnode ImgCallback: "
+         << " frame: " << ++seq_num_ << " timestamp: ";
```

Comment:
This ++seq_num_ is duplicated. The one below is desired

Reply:
thanks. Fixed.